[2021-09-23 22:54:54,206] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-09-01T00:00:00+00:00 [queued]>
[2021-09-23 22:54:54,217] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-09-01T00:00:00+00:00 [queued]>
[2021-09-23 22:54:54,218] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-09-23 22:54:54,218] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-09-23 22:54:54,218] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-09-23 22:54:54,243] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): runme_2> on 2021-09-01T00:00:00+00:00
[2021-09-23 22:54:54,247] {standard_task_runner.py:52} INFO - Started process 4804 to run task
[2021-09-23 22:54:54,255] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', '2021-09-01T00:00:00+00:00', '--job-id', '139', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/sandbox/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp9pruonxs', '--error-file', '/tmp/tmp5thgk4d_']
[2021-09-23 22:54:54,258] {standard_task_runner.py:77} INFO - Job 139: Subtask runme_2
[2021-09-23 22:54:54,332] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_2 2021-09-01T00:00:00+00:00 [running]> on host airflowvm
[2021-09-23 22:54:54,410] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2021-09-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-01T00:00:00+00:00
[2021-09-23 22:54:54,411] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-23 22:54:54,412] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo "example_bash_operator__runme_2__20210901" && sleep 1']
[2021-09-23 22:54:54,417] {subprocess.py:75} INFO - Output:
[2021-09-23 22:54:54,419] {subprocess.py:79} INFO - example_bash_operator__runme_2__20210901
[2021-09-23 22:54:55,423] {subprocess.py:83} INFO - Command exited with return code 0
[2021-09-23 22:54:55,450] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20210901T000000, start_date=20210923T225454, end_date=20210923T225455
[2021-09-23 22:54:55,497] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-09-23 22:54:55,518] {local_task_job.py:151} INFO - Task exited with return code 0
